associate-software-engineer/fc-1/javascript/modulo-7/topico-1/index.js

```diff
@@ -0,0 +1,34 @@
+const title = document.getElementById("tituloPrincipal");
+title.textContent = "Aooba";
+
+const paragraphs = document.getElementsByTagName("p");
+for (const p of paragraphs) {
+  p.style.color = "#3346FF";
+}
+
+const highlightedElement = document.getElementsByClassName("destaque");
+for (const alterBackground of highlightedElement) {
+  alterBackground.style.backgroundColor = "#ffff00";
+}
+for (const alterContent of highlightedElement) {
+  alterContent.textContent = "Texto alterado";
+}
```

Comment:
mesma coisa. alterContent é a ação que vc quer que faça eu preciso que vcs prestem atenção nisso. O que precisa ficar gravado na cabeça é que toda variável representa no nome o seu conteúdo guardado. E cada função representa a ação que vai ser realizada, logo alterContent seria o nome de uma função. 

Eu to pegando nesse ponto pq esse conceito é muito importante, não é só questão do nome da variável.